docs/product-highlights-and-benefits/sharing-to-and-from-github-gists.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: Sharing To & From GitHub Gists
+description: GitHub Gists are a great way to share code with a group of people for public visibility, and integrate your snippets with your GitHub repositories. Pieces makes it easy to update your Gist repository by enabling you to share saved code snippets within Pieces as Gists, right from the desktop app.
```

Comment:
this is what is going to be used as the meta description so we should just make sure that it is below the appropriate length